### PR TITLE
Define a checkout (ecommerce) area

### PR DESCRIPTION
**Pull request by Igor-J86**
Restructuring the content with this PR, where we are defining a dedicated checkout area, or rather ecommerce area. Here we can present useful resources for ecommerce, as the "Checkout guide", "Checkout request builder" and "Product page widget".

The name that is given now - **Ecommerce solutions**
This is up for discussion.

Missing in this PR:
- Aliases for old urls
- Making the entrance for "Ecommerce solutions" more prominent on the frontpage

Skjermbilde 2022-10-27 kl  15 40 48

Skjermbilde 2022-10-27 kl  15 41 22

### CODE REVIEW

File: content/api/e-commerce-solutions/_index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: E-commerce solutions
+layout: api
+notanapi: true
+menu:
+  api:
+    identifier: e-commerce-solutions
+    title: E-commerce solutions
+    url: e-commerce-solutions
+weight: 4
+aliases:
```
Comment (halvorsanden) [inline content/api/e-commerce-solutions/_index.html:11]: Can be removed, unless we want the old `ecommerce` alias here. I doubt anyone uses that anymore though.

Comment (Igor-J86) [reply content/api/e-commerce-solutions/_index.html:11]: Right, I think it's ok to remove it too..

File: content/api/e-commerce-solutions/_index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+<section class="dev-docscontent__section">
+  <p class="text-heading fw600">
+    Build engaging shopping experience in your webshop by offering customers´ delivery time directly on the product page with the new <a href='{{< relref "product-page-widget" >}}'>Product Page Widget</a>. You will also find useful resources in the <a href='{{< relref "checkout-guide-norway" >}}'>Checkout guide Norway</a> and the <a href="https://www.mybring.com/shipping-guide/integration/checkout" target="_blank">Checkout Request Builder</a>, so you can make sure your customers are met with great delivery options in your webshop's checkout.
```
Comment (halvorsanden) [inline content/api/e-commerce-solutions/_index.html:16]:
```suggestion
    Build engaging shopping experiences in your webshop by following recommended checkout API implementations in the <a href='{{< relref "checkout-guide-norway" >}}'>Checkout Guide Norway</a> and the <a href="https://www.mybring.com/shipping-guide/integration/checkout" target="_blank" rel="noopener noreferrer">Checkout Request Builder</a>. You can also show delivery times already on product pages with the <a href='{{< relref "product-page-widget" >}}'>Product Page Widget</a>.
```

File: content/api/e-commerce-solutions/_index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  <div class="flex flex-wrap gas mvl">
+    <a href='{{< relref "product-page-widget" >}}' class="mb-card bg-green1 pam flex flex-dir-col ecom-entrance">
+      <div class="flex mlxs align-ic">
+        <span
+          data-mybicon="mybicon-sidebar"
+          data-mybicon-class="icon-ui--green-dark mrs"
+          data-mybicon-width="19"
+          data-mybicon-height="19"
+        ></span>
+        <h3 class="text-heading fwn">Product page widget</h3>
```
Comment (halvorsanden) [inline content/api/e-commerce-solutions/_index.html:27]: These headings should be h2 according to the document structure. But that will make them very big if we also remove the text-heading class like on the front page. Maybe switch to h2 and just keep the class?

Comment (Igor-J86) [reply content/api/e-commerce-solutions/_index.html:27]: Good catch! Switching them to h2.. 🙂

File: content/api/e-commerce-solutions/product-page-widget/_index.html [hunk 4]
```diff
@@ -2,17 +2,21 @@
...
   <p>The product page widget is used on product pages to inform how quickly we can deliver the product to the customer.</p>
   <p>Our e-commerce widgets are fully integrated with relevant APIs, and represent a simplified way of presenting our services on your webshop. Using our widgets, you ensure that the way you present Posten’s delivery options is in line with our best practice recommendations.</p>
   <p>
     Implementation guide is available with the configuration of the widget on mybring.
   </p>
   <p><a href="https://www.mybring.com/shipping-guide/widgets/ps">Configure your widget (requires you to log in)</a></p>
-  <img src="/api/product-page-widget/product-page-widget.png" alt="Product page widget image"/>
-</section>
-</main>
+  <img src="/api/e-commerce-solutions/product-page-widget/product-page-widget.png" class="maxw40r" alt="Product page widget image"/>
```
Comment (halvorsanden) [inline content/api/e-commerce-solutions/product-page-widget/_index.html:21]:
```suggestion
  <img src="/api/e-commerce-solutions/product-page-widget/product-page-widget.png" class="maxw24" alt=""/>
```
Just setting the size closer to what the widget is, and the alt text can be blank as it’s mostly decorative.

File: content/api/e-commerce-solutions/product-page-widget/_index.html [hunk 5]
```diff
@@ -2,17 +2,21 @@
...
 ---
 
-<main id="main" class="maxw48r w100p mvl phl jobs">
 <section class="dev-docscontent__section">
   <p>The product page widget is used on product pages to inform how quickly we can deliver the product to the customer.</p>
   <p>Our e-commerce widgets are fully integrated with relevant APIs, and represent a simplified way of presenting our services on your webshop. Using our widgets, you ensure that the way you present Posten’s delivery options is in line with our best practice recommendations.</p>
   <p>
     Implementation guide is available with the configuration of the widget on mybring.
   </p>
   <p><a href="https://www.mybring.com/shipping-guide/widgets/ps">Configure your widget (requires you to log in)</a></p>
```
Comment (halvorsanden) [inline content/api/e-commerce-solutions/product-page-widget/_index.html:20]:
```suggestion
  <p>The product page widget is used on product pages to inform how quickly we can deliver the product to the customer.</p>
  <p>Our e-commerce widgets are fully integrated with relevant APIs and represent a simplified way of presenting our services on your webshop. Using our widgets, you ensure that you present Posten’s delivery options in line with our best practice recommendations.</p>
  <p>An implementation guide is available with the <a href="https://www.mybring.com/shipping-guide/widgets/ps">widget configuration in Mybring</a> (requires log in).</p>
```

**Review by halvorsanden — APPROVED** [review]
Feel free to merge after reviewing the comments. 🎉

File: content/api/e-commerce-solutions/_index.html (hunk 2 above)
Comment (Igor-J86) [reply content/api/e-commerce-solutions/_index.html:15]: Nice! 😃 Thanks!

**Pull request merged**